src/Donations/Endpoints/DonationUpdateAttributes/DonationUpdateAttribute.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Give\Donations\Endpoints\DonationUpdateAttributes;
+
+use Give\Donations\Models\Donation;
+
+abstract class DonationUpdateAttribute
+{
+    /**
+     * @return string
+     */
+    abstract public static function getId(): string;
+
+    /**
+     * @return array
+     */
+    abstract public static function getDefinition(): array;
+
+    /**
+     * @param          $value
+     * @param Donation $donation
+     *
+     * @return Donation|bool
+     */
+    public static function update($value, Donation $donation)
+    {
+        return false;
+    }
```

Comment:
I think this can just be an abstract method as nothing really needs to be returned. The model can simply be passed and if its mutated its mutated.